### PR TITLE
remove MenuContainer

### LINKED ISSUE #2171: remove menu container
all of our menus use the new MenuWithShortcuts component, which is keyboard accessible & shows focus, animations, etc. it works great. There are 3 components that don't use it:

Acceptance criteria
- [x] dropdowninput.js refactored
- [x] serviceDropdownInput refactored
- [x] TaskColumn refactored
- [x] react-addons-css-transition-group removed
- [x] Menu, MenuContainer, and MenuItem removed

effort: 10.5 points

### PR DESCRIPTION
**Pull request by mattkrick**
All menus are now keyboard accessible & have nice transitions & we no longer get that nasty warning in dev. fixes #2171

TEST
- on New Team view
  - [ ] can change organization, the dropdown looks good
  - [ ] creating a new team makes the org dropdown disabled

- on GitHub & Slack add service view
  - [ ] can click the dropdown to see all github repos/slack channels
  - [ ] the dropdown takes up the full width & is clickable
  - [ ] can add repo/slack channel integration

- on My Dashboard
  - [ ] can add a new task & select the team you want the task to appear on (must have more than 1 team when testing)

### CODE REVIEW

**Comment by mattkrick** [comment]
@jordanh ready for review!

**Comment by jordanh** [comment]
# Test Cycle 1

- on New Team view
  - [ ] can change organization, the dropdown looks good
  - [ ] creating a new team makes the org dropdown disabled

- on GitHub & Slack add service view
  - [ ] can click the dropdown to see all github repos/slack channels
  - [ ] the dropdown takes up the full width & is clickable
  - [ ] can add repo/slack channel integration

- on My Dashboard
  - [ ] can add a new task & select the team you want the task to appear on (must have more than 1 team when testing)

File: src/universal/components/LoadableNewTeamOrgDropdown.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import LoadableLoading from 'universal/components/LoadableLoading'
+
+const LoadableNewTeamOrgDropdown = Loadable({
+  loader: () =>
+    import(/* webpackChunkName: 'NewTeamOrgDropdown' */
+      'universal/components/NewTeamOrgDropdown'),
+  loading: (props) => (
+    <LoadableLoading {...props} height={DEFAULT_MENU_HEIGHT} width={DEFAULT_MENU_WIDTH} />
+  ),
+  delay: HUMAN_ADDICTION_THRESH,
```
Comment (jordanh) [inline src/universal/components/LoadableNewTeamOrgDropdown.js]: This constant name tho 🤣

**Comment by jordanh** [comment]
All tests passed. @mattkrick see notes above. Merge when ready!

**Comment by mattkrick** [comment]
created a new issue to fix the sort,
this one had a conflict with master that looks like it explains #2319

**Pull request merged**